### PR TITLE
feat: Landscape profile image

### LINKED ISSUE #570: Landscape Profile: Profile picture add/update
## Description
- Add profile picture update/edit form
- Use crop tool

## Additional context

Crop plugin: https://github.com/mosch/react-avatar-editor
Figma: https://www.figma.com/file/Uv20vRx7hDmeDw4c5dOhiY/Create%2FUpdate-Landscape-Profile_V2-R1.1?node-id=0%3A1

### LINKED ISSUE #573: Landscape Profile: Landscape Profile page
## Description
- Create landscape profile page
- Add link to full profile in About Landscape card

## Additional context

Figma: https://www.figma.com/file/Uv20vRx7hDmeDw4c5dOhiY/Create%2FUpdate-Landscape-Profile_V2-R1.1?node-id=0%3A1

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Landscape Profile Image update form
- Landscape Profile Image display in profile page

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added

### Related Issues
Fixes #570 

### Dependencies
- [terraso-backend#219](https://github.com/techmatters/terraso-backend/pull/219)

### Verification steps

Screen Shot 2022-11-09 at 3 22 03 PM
Screen Shot 2022-11-09 at 3 21 46 PM
Screen Shot 2022-11-09 at 3 14 09 PM

### CODE REVIEW

**Review by david-code — COMMENTED** [review]
Looks good to me! I will let Paul take a look as well.

File: src/landscape/components/LandscapeForm/ProfileImageStep.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    .then(blob => {
+      return new Promise((resolve, reject) => {
+        const reader = new FileReader();
+        reader.onloadend = () => {
+          resolve(reader.result);
+        };
+        reader.onerror = () => reject(reader.error);
+        reader.readAsDataURL(blob);
+      });
+    });
```
[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/ProfileImageStep.js:64]: @david-code Thanks David, and from your comment I realized that there is more code that can be reused between openImage and openFile

File: src/landscape/components/LandscapeForm/ProfileImageStep.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+            image={image}
+            width={size.width}
+            height={size.height}
+            border={editorBorder}
+            color={[255, 255, 255, 0.6]} // RGBA
+            scale={zoom}
+            rotate={0}
+            onImageReady={() => {
+              const width = containerRef.current.offsetWidth - editorBorder * 2;
+              const height = (width * 9) / 16;
```
[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/ProfileImageStep.js:130]: @david-code I came from the Figam designs, and I agree I will move this to a constant

File: src/landscape/components/LandscapeProfile/ProfileCard.js [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+  Stack,
+  Typography,
+} from '@mui/material';
 
 import { countryNameForCode } from 'common/utils';
 import Restricted from 'permissions/components/Restricted';
 
 import { getTermLabel } from 'taxonomies/taxonomiesUtils';
 
+import profileImageDefault from './profile-image-default.jpg';
```
Comment (paulschreiber) [inline src/landscape/components/LandscapeProfile/ProfileCard.js:23]: question: can this go in `public` with the other images?

File: src/landscape/components/LandscapeForm/ProfileImageStep.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+const ProfileImage = props => {
+  const { t } = useTranslation();
+  const {
+    field: { value, onChange },
+  } = props;
+  const [error, setError] = useState();
+  const [image, setImage] = useState();
+  const [zoom, setZoom] = useState(1);
+  const [size, setSize] = useState({ width: 418, height: 235 });
```
Comment (paulschreiber) [inline src/landscape/components/LandscapeForm/ProfileImageStep.js:69]: suggestion: put these values in constants.

File: src/landscape/components/LandscapeProfile/ProfileCard.js [hunk 5]
```diff
@@ -91,6 +100,59 @@ const ProfileField = props => {
...
+          >
+            {t('landscape.profile_profile_card_profile_image_update')}
+          </Button>
+        </Stack>
+      )}
+      <img
+        src={
+          hasProfileImage
+            ? landscape.profileImage
+            : '/landscape/profile-image-default.jpg'
```
Comment (paulschreiber) [inline src/landscape/components/LandscapeProfile/ProfileCard.js:144]: suggestion: put this in a constant.

**Review by paulschreiber — CHANGES_REQUESTED** [review]

**Pull request merged**